src/get-port.ts

```diff
@@ -73,12 +73,16 @@ export async function getPort(
       _generateRange(...options.alternativePortRange),
       options.host,
     );
-    _log(
-      options.verbose,
-      `Unable to find an available port (tried ${options.alternativePortRange.join(
-        "-",
-      )} ${_fmtOnHost(options.host)}). Using alternative port ${availablePort}`,
-    );
+    if (portsToCheck.length > 0 && availablePort) {
+      _log(
+        options.verbose,
+        `Unable to find an available port (tried ${portsToCheck.join(
+          "-",
+        )} ${_fmtOnHost(
+          options.host,
+        )}). Using alternative port ${availablePort}`,
```

Comment:
Shall we move `availablePort` check part to partially remove last part of the message?